backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/model/InterfaceWrapperHelper.java

```diff
@@ -1646,7 +1646,7 @@ public static Evaluatee getEvaluatee(final Object model)
 
 	/**
 	 * @param modelClass
-	 * @return immutable list of column names of modelClass's table
+	 * @return immutable list of physical column names of modelClass's table
 	 */
 	public static Set<String> getModelColumnNames(final Class<?> modelClass)
```

Comment:
please rename to `getModelPhysicalColumnNames()`

Reply:
done